tests/test_measure.cpp

```diff
@@ -10,15 +10,17 @@
 // xdg test includes
 #include "mesh_mock.h"
 #include "util.h"
+#include "xdg/embree_ray_tracer.h"
 
 using namespace xdg;
 
 TEST_CASE("Test Mesh Mock")
 {
   std::shared_ptr<MeshManager> mm = std::make_shared<MeshMock>();
+  std::shared_ptr<RayTracer> rti = std::make_shared<EmbreeRayTracer>();
   mm->init(); // this should do nothing, but its good practice to call it
```

Comment:
```suggestion
  mm->init(); // this should do nothing, but it's good practice to call it
```

Pre-existing but thank you in advance for indulging my OCD